shell.go

```diff
@@ -40,12 +40,11 @@ var Shells = map[string]Shell{
 	powershell: {
 		Command: []string{
 			"powershell",
-			"-Login",
 			"-NoLogo",
 			"-NoExit",
 			"-NoProfile",
 			"-Command",
-			`Set-PSReadLineOption -HistorySaveStyle SaveNothing; Function prompt { Write-Host -ForegroundColor Blue -NoNewLine '>'; return ' ' }`,
+			`Set-PSReadLineOption -HistorySaveStyle SaveNothing; function prompt { Write-Host '>' -NoNewLine -ForegroundColor Blue; return ' ' }`,
```

Comment:
is this necessary too? why?

Reply:
After removing `-Login` command `vhs` stacks on `Set Shell powershell` instruction. I remove `-Command` parameter and all was fine. I checked how to rewrite default prompt in PS and write this `-Command` based on [this instruction](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_prompts?view=powershell-7.3)

Reply:
aahh cool, thanks for the info!